### DISCUSSION TITLE
Vue3 simple login page "supabase.auth.signIn is not a function"

### DISCUSSION THREAD

**Discussion by ashbomb328** [supabase/supabase#13762] [Questions]
I cannot get my Login page to work, and I keep getting the error supabase.auth.signIn is not a function. I have re-installed supabase and checked versions so many times. I am using Vue3 and supabase 2.20. here is my applogin.vue-
"export default {
  name: 'AppLogin',
  data() {
    return {
      email: '',
      password: ''
    }
  },
  methods: {
    async login() {
      try {
        const { user, error } = await supabase.auth.signIn({
          email: this.email,
          password: this.password
        })

        if (error) {
          console.error('Error logging in:', error.message)
        } else {
          console.log('User logged in:', user)
        }
      } catch (error) {
        console.error('Error logging in:', error.message)
      }
    }
  }
}
</script>
" and here is my supabase.js-
// src/supabase.js
import { createClient } from '@supabase/supabase-js';

const supabaseUrl = 'xx';
const supabaseKey = 'xx';

export const supabase = createClient(supabaseUrl, supabaseKey);

export default supabase

I am also able to show any other part of my code. Is this a backend problem or a code problem?

**Comment by gregnr** [comment]
Hey @ashbomb328! Sorry to hear you're experiencing this. Just to confirm, are you importing `supabase` in your `applogin.vue` component?
```js
import { supabase } from '../supabase'
```
^ assuming `applogin.vue` lives under `src/components/`
